src/pages/profile/index.tsx

```diff
@@ -78,6 +78,7 @@ const Profile = () => {
     insertUserIfNeeded().then(() => {
       upsertAgency({ variables: { email: user?.email, agency } });
     });
+    if (!onboarding) router.reload();
```

Comment:
```suggestion
      upsertAgency({ variables: { email: user?.email, agency } });
      if (!onboarding) router.reload();
    });
```

Reply:
Otherwise, when I try to change my agency, the page reloads before the request is finished and changes are not saved successfully.